### DISCUSSION TITLE
[2.3.2] Tuning assist

### DISCUSSION THREAD

**Discussion by RagnarSTS** [Security-Onion-Solutions/securityonion#1894] [Unsupported Versions] [answered]
***Fix -- YAML doesn't like tabs.  ***
*****************************************

Running 2.3.2 from an ISO install with a standalone deployment.
Attempting pass #1 on tuning out noisy alerts is giving me fits.
Following instructions from [the docs](https://docs.securityonion.net/en/2.3/managing-alerts.html)
leads me to editing `/opt/so/saltstack/local/pillar/minions/xxxxx.sls`  
Adding a single "  - nnnnn" sid to disable under 
`idstools:
  sids:
    disabled:
      - xxxxxx`
winds up with the following error upon running salt state.apply idstools (without full filename ie seco2_standalone):
```
Data failed to compile:
Pillar failed to render with the following messages:
Rendering SLS 'minions.seco2_standalone' failed. Please see master log for details.
ERROR: Minions returned with non-zero exit code
```
(or WITH full filename ie seco2_standalone.sls):
```
No minions matched the target. No command was sent, no jid was assigned.
ERROR: No return received
```
I guess I'm a little confused in the documentation, and I'm not sure which error, if any, is better than the other.

I know if I leave it unable to compile and restart the services or the entire host, it doesn't come up correctly at all.  I am also unclear about the 'master log' ...  there is a /var/log/salt/master  ... but it is a completely empty file.

**Comment by greatapoc** [comment, marked as answer]
Make sure you're using spaces and not tabs for indenting. Your first command is correct too, it should be salt seco2_standalone state.apply idstools.

The master log it refers to is /opt/so/log/salt/master and it should reveal the reason.

**Reply by RagnarSTS** [reply]
omg...I'm going to go outside and throw things now.
This world where tabs are not acceptable just might lead me to my death.
Confirm spaces a dash and a space with the sid listed indeed salts properly without error.

**Reply by greatapoc** [reply]
Welcome to YAML! Much fun awaits.
